.github/workflows/check-cves.yaml

```diff
@@ -0,0 +1,27 @@
+name: check cves
+
+on:
+  push:
```

Comment:
Can you add a periodic trigger? 

And an accompanying 'keepalive' job?

Reply:
Added here: https://github.com/canonical/robotics-action-workflows-tests/pull/5